config/locales/server.en.yml

```diff
@@ -1143,6 +1143,7 @@ en:
     log_search_queries: "Log search queries performed by users"
     search_query_log_max_size: "Maximum amount of search queries to keep"
     search_query_log_max_retention_days: "Maximum amount of time to keep search queries, in days."
+    ignore_accents: "Ignore accents when searching for text."
```

Comment:
one last thing... let's call this `search_ignore_accents` ... far clearer name cause this is very search specific. Once done feel free to merge